### ISSUE TITLE
when hitting "stop temporarily" you return to first welcome page

### ISSUE THREAD

**Issue by SanVerhavert** [d-pac/d-pac.client#76]
```json
{"now":"2015-05-28T11:37:38.131Z","url":"http://staging.tool.d-pac.be/assessors/#welcome","version":"0.4.0 (659)","request":{"url":"/api/timelogs/5566fdbfb4738fcd435a31d8","uuid":"26830466-e765-45f0-ba05-1603c6328199"},"userAgent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.81 Safari/537.36"}
```
![image](https://cloud.githubusercontent.com/assets/9549353/7859209/05bd306e-053f-11e5-98d6-3fb238ecacbd.png)

**Comment by SanVerhavert** [comment]
possibly related to issue #71

**Comment by creynders** [comment]
Fixed in 9ff206e004d38cd73ce3de9f38ff1aa73e909a3e
